widget/embedded/src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,41 @@
+import type { PropTypes } from './Filter.type';
+
+import { FilterIcon, Popover } from '@rango-dev/ui';
+import React, { useState } from 'react';
+
+import { getContainer } from '../../utils/common';
+
+import { Badge, Button, IconContainer } from './Filter.style';
+import { FilterContent } from './FilterContent';
+
+export function Filter(props: PropTypes) {
+  const [open, setOpen] = useState(false);
+  return (
+    <div>
+      <Popover
+        open={open}
+        align="end"
+        onOpenChange={(open) => setOpen(open)}
+        container={getContainer()}
+        content={
+          <FilterContent
+            filterBy={props.filterBy}
+            onClickItem={(id) => {
+              props.onClickItem(id);
+              setOpen(false);
+            }}
+          />
+        }>
+        <Button
```

Comment:
This might be mistaken with the general Button component. please use a specific name